osu.Game/Rulesets/Edit/HitObjectComposer.cs

```diff
@@ -27,6 +29,9 @@ public abstract class HitObjectComposer : CompositeDrawable
         private RulesetContainer rulesetContainer;
         private readonly List<Container> layerContainers = new List<Container>();
 
+        private readonly Bindable<WorkingBeatmap> beatmap = new Bindable<WorkingBeatmap>();

```

Comment:
I think eventually we're going to want this to not bind from the osuGame.Beatmap, hence why I've created a local bindable. Makes it pretty awkward to test things when tests modify osuGame.Beatmap...